language_container/exasol_sagemaker_extension_container/flavor_base/release/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM exasol/script-language-container:template-Exasol-all-python-3.10-build_run_BUXX2FHQPEBXZNMS6RQRXPKRMKZREERZ5HIRVNXTV56DLNXXTGQA
+FROM exasol/script-language-container:template-Exasol-all-python-3.10-release_BFRSH344TDRPT7LK2FBOJK4KBIDW6A253FFPYEUYT4O2ERFMTCNA
```

Comment:
This sounds like a very magic number. Can you explain (or add a comment into the file)?
* Where does the number come from?
* How is it generated (maybe omitting details)?

Reply:
A simple answer is that the magic number is a hash that identifies the base image build.
To give a more meaningful answer, I need to take my knowledge about this subject to a different level. This will be a good idea, but I probably should not book a time for that in the scope of this ticket.

Reply:
Torsten explained: The ID is retrieved from docker hub